frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -117,9 +128,16 @@ export const VisualizationDirtyState = ({
   hidden,
 }) => (
   <div
-    className={cx(className, "Loading flex flex-column layout-centered", {
-      "Loading--hidden pointer-events-none": hidden,
-    })}
+    className={cx(
+      className,
+      QueryBuilderS.Loading,
+      CS.flex,
+      "flex-column layout-centered",
```

Comment:
I just noticed we already have `CS.layoutCentered`, but this could be worked on in another PR that changes this class globally.